TrackingTools/TransientTrack/interface/TransientTrackBuilder.h

```diff
@@ -13,6 +13,10 @@
    * directory.
    */
 
+namespace reco {
+  constexpr float defaultInvalidTrackTimeReso = 0.350f;
```

Comment:
can this be a member of the `TransientTrackBuilder` instead?
if the idea is to introduce a more generic constant not related to details of `TransientTrackBuilder`, this should be defined in a different file

Reply:
@slava77 I suggested this solution to @kdeleo as the minimal change of existing behaviour that allows us nevertheless to have a more transparent code, without magic numbers duplicated or triplicated in different places, or even worse, similar numbers meant to be the same used in different places, see

https://github.com/cms-sw/cmssw/blob/be8e1ec40b9812e563a14867d8c92c8c46828ae6/RecoVertex/PrimaryVertexProducer/src/DAClusterizerInZT_vect.cc#L241

Looking back at the history of the addition of this constant to the ```TransientTrackBuilder``` , it arrives well before (i.e. 2016 - 2017) the current behaviour of the ```TrackExtenderWithMTD``` and of the import of this information into particle flow. And it is clearly only in the ```PrimaryVertexProducer``` as far as I can see. 

A more coherent coding with the way the "no time info" case is propagated now to tracks would be to set in the ```TransientTrackBuilder``` the time error to -1, and adapt the vertex producer code accordingly. As far as we can see with @kdeleo the value is used just as a threshold to trigger some decision.

Reply:
Actually what I say is not fully true. The time error in the ```TransientTrack``` is also used, in its current definition, in the ```AdaptiveVertexFinder``` package. While updating that code to use a new definition of the time error could be relatively simple, checking that the net result in secondary vertex handling is unchanged is definitely outside the scope of this PR. 

So if @slava77 has a strong feeling about this issue, we can turn the constant into a private member of the class, initialised by default to the current value, and add a getter method to use it consistently outside the ```TransientTrack``` itself. I would consider preferable to coherently update the treatment of the case of no time everywhere to go beyond this ad hoc constant at some point, and the proposed solution of this PR maintains for the time being the existing functionality in a slightly more clear way.

Reply:
> we can turn the constant into a private member of the class, initialised by default to the current value

A static or constexpr public constant should also be OK; if the value is really attached to the `TransientTrack` details. `reco::default...` seems to imply something pretty general